client/src/components/loading/component.tsx

```diff
@@ -3,7 +3,7 @@ import type { LoadingProps } from './types';
 
 export const Loading: React.FC<LoadingProps> = ({ className }: LoadingProps) => (
   <svg
-    className={classNames('animate-spin -ml-1 mr-3 h-5 w-5 stroke-none', className)}
```

Comment:
Removal of margins is because it makes it impossible to set them on a per-case basis.